packages/grafana-ui/src/components/DataSourceSettings/AlertingSettings.tsx

```diff
@@ -19,7 +19,12 @@ export function AlertingSettings<T extends AlertingConfig>({ options, onOptionsC
       <div className="gf-form-group">
         <div className="gf-form-inline">
           <div className="gf-form">
-            <InlineField labelWidth={26} label="Manage alerts via Alerting UI" disabled={options.readOnly}>
```

Comment:
Seeing this bug when using the new typeface. 

<img width="414" alt="Screenshot 2023-04-11 at 13 14 44" src="https://user-images.githubusercontent.com/5264102/231239460-04eca984-3975-4a49-9c52-f887034d1320.png">